test/helloworld_test.go

```diff
@@ -53,20 +51,21 @@ func TestRunHelloworld(t *testing.T) {
 	}
 }
 
-func TestBuildHelloworldWithVendor1(t *testing.T) {
-	UseApp(HelloworldAppName)
-	util.RunCmd("go", "mod", "vendor")
-	RunInstrument(t, "-debuglog")
-}
+// FIXME: Support vendor mode
+// func TestBuildHelloworldWithVendor1(t *testing.T) {
```

Comment:
If we don't support vendor mod, we had better to state in the document

Reply:
Added